apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -74,7 +74,7 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
         <h1 className="text-4xl font-semibold">Documents</h1>
 
         <div className="-m-1 flex flex-wrap gap-x-4 gap-y-6 overflow-hidden p-1">
-          <Tabs defaultValue={status} className="overflow-x-auto">
+          <Tabs defaultValue={status} value={status} className="overflow-x-auto">
```

Comment:
We could remove defaultValue={status} here and it would work just fine.

Reply:
Feel free to do so :)